source/event_hub.ts

```diff
@@ -606,20 +606,24 @@ export class EventHub {
    * Publish reply event.
    * @param  {Object} sourceEventPayload Source event payload
    * @param  {Object} data        Response event data
-   * @param  {Object} [source]    Response event source information
+   * @param  {Object} source      Deprecated value - will be removed in a later version
    */
   publishReply(
     sourceEventPayload: EventPayload,
     data: Data,
     source: Data | null = null
   ): Promise<string> {
-    const replyEvent = new Event("ftrack.meta.reply", {
-      ...data,
-      target: `id=${sourceEventPayload.source.id}`,
-      inReplyToEvent: sourceEventPayload.id,
-      source: source ?? data.source,
-    });
-
+    const replyEvent = new Event(
+      "ftrack.meta.reply",
+      {
+        ...data,
```

Comment:
Nit: Any reason for doing a shallow copy of data rather than just passing it through?

Reply:
No